src/index.js

```diff
@@ -17,16 +19,18 @@ messages
 } = state;
 
 ReactDOM.render(
-  <App 
+  <Provider store = {store} >
+  <App
```

Comment:
Now that you're using redux, you won't have to pass the props into each component from App. Remember, we pass props directly into the components themselves using containers and mapStateToProps. Doing it again here is redundant.